phpcs-report.xml

```diff
@@ -0,0 +1,1240 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be ignored.

Reply:
@bjorvack can you fix this please

Reply:
@tijsverkoyen fixed fbf6fb17052ca79eda2e13a4dd3b5295c08a8682